staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -80,6 +86,10 @@ type Config struct {
 	// and metrics.
 	GroupResource schema.GroupResource
 
+	// EventsHistoryWindow specifies minimum history duration that storage is keeping.
+	// If lower than <DefaultEventFreshDuration>, the cache creation will fail.
```

Comment:
nit: what the syntax to reference identifiers in comments? Above you use backticks.

Reply:
Dropped the backticks and the angle brackets to make referencing the identifiers consistent with other references in this file.

Reply:
is there some standard in kube (not sure I ever saw `<>`) or even in Golang?

Reply:
I see https://tip.golang.org/doc/comment#doclinks suggests using square brackets - though I'd prefer to stick to the convention used throughout in this file for consistency's sake.

The `<>` angle brackets I used are an artifact of my copy-pasting this comment from somewhere else :)